src/cli/parse-flags.ts

```diff
@@ -42,8 +42,8 @@ export const parseFlags = (args: string[], sys?: CompilerSystem): ConfigFlags =>
 };
 
 const parseArgs = (flags: any, args: string[], knownArgs: string[]) => {
```

Comment:
Although this function is relatively self-explanatory based on it's name, can we add a JSDoc to it while we're in here?
```suggestion
/**
 * A short description of the function, ideally 1-2 sentences
 * @param flags a short description of this argument
 * @param args  a short description of this argument 
 * @param knownArgs a short description of this argument
 */
const parseArgs = (flags: any, args: string[], knownArgs: string[]) => {
```

Reply:
Agreed, that caught my eye too - probably better for another day